lib/http-client-base.ts

```diff
@@ -164,7 +165,7 @@ export class HttpClient {
         err,
       );
     } else if (err.code) {
-      return new RequestError(err.message, err.code, err.response.data.statusMessage || "", err);
+      return new RequestError(err.message, err.code, "", err);
```

Comment:
Sorry I don't get it why we have to assign empty string to `detailErrorMessage`?
if it cause a problem then wen can do like below.

```
return new RequestError(err.message, err.code, err.response.data.statusMessage ? err.response.data.statusMessage :"", err);
```

Reply:
In this block, since the `err` object does not have a `response` property, we are returning an empty string.